performance/SwiftUIPerformanceQuickstart/PerformanceExample/Design/SYMBOLS.md

```diff
@@ -0,0 +1,371 @@
+# Performance Monitoring Quickstart SwiftUI Symbols
+
+## Table of Contents
+- [PerformanceExampleApp](#performanceexampleapp)
+    - [init](#init)
+    - [body](#body)
+- [Process](#process)
+    - [status](#status)
+    - [image](#image)
+    - [saliencyMap](#saliencymap)
+    - [uploadSucceeded](#uploadsucceeded)
+    - [categories](#categories)
+    - [context](#context)
+    - [isRunning](#isrunning)
+    - [precision](#precision)
+    - [site](#site)
+    - [updateStatusAsync](#updatestatusasync)
+    - [updateImageAsync](#updateimageasync)
+    - [updateSaliencyMapAsync](#updatesaliencymapasync)
+    - [downloadImageAsync](#downloadimageasync)
+    - [classifyImageAsync](#classifyimageasync)
+    - [generateSaliencyMapAsync](#generatesaliencymapasync)
+    - [uploadSaliencyMapAsync](#uploadsaliencymapasync)
+    - [updateStatus](#updatestatus)
+    - [updateImage](#updateimage)
+    - [updateSaliencyMap](#updatesaliencymap)
+    - [downloadImage](#downloadimage)
+    - [classifyImage](#classifyimage)
+    - [generateSaliencyMap](#generatesaliencymap)
+    - [uploadSaliencyMap](#uploadsaliencymap)
+    - [makeTrace](#maketrace)
+- [ProcessDerivatives](#processderivatives)
+    - [ProcessTask](#processtask)
+    - [ProcessStatus](#processstatus)
+        - [text](#text)
+        - [view](#view)
+- [MainView](#mainview)
+    - [process](#process-1)
+    - [body](#body-1)
+- [DownloadView](#downloadview)
+    - [process](#process-2)
+    - [body](#body-2)
+- [ClassifyView](#classifyview)
+    - [process](#process-3)
+    - [body](#body-3)
+- [SaliencyMapView](#saliencymapview)
+    - [process](#process-4)
+    - [body](#body-4)
+- [UploadView](#uploadview)
+    - [process](#process-5)
+    - [body](#body-5)
+
+## PerformanceExampleApp
+```swift
+@main
+struct PerformanceExampleApp: App
+```
+main point of entry into app
+
+### init
```

Comment:
optional: for these headers, you could make them inline code blocks
> ### `init`
> ```swift
> init()
> ```
> configures the FirebaseApp

Reply:
Done ✅